src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -223,15 +226,15 @@ public override async Task<P.GetInstanceResponse> GetInstance(P.GetInstanceReque
                 return CreateGetInstanceResponse(state, request);
             }
 
-            public override async Task<P.QueryInstancesResponse> QueryInstances(P.QueryInstancesRequest request, ServerCallContext context)
+            public async override Task<P.QueryInstancesResponse> QueryInstances(P.QueryInstancesRequest request, ServerCallContext context)
             {
-                OrchestrationQuery query = ProtobufUtils.ToOrchestrationQuery(request);
+                var query = ProtobufUtils.ToOrchestrationQuery(request);
```

Comment:
Why the switch to `var`? My preference is that variable types be explicit.

Reply:
I was following our `.editorconfig` configuration here.